pcs_detection/src_python/pcs_detection/utils.py

```diff
@@ -42,7 +42,7 @@ def get_labels_from_xml(label_path):
     
     # name of the outer directory this label corresponds to
     # allows for images in different directories to have the same filename 
-    dataset_name = label_path.split('/')[4]
+    dataset_name = label_path.split('/')[-2]
```

Comment:
Is -2 always correct?

Reply:
Should be with the current convention. I typically like to leave labels.xml in a directory above the actual images

Reply:
If that is a requirement we should put that in the readme.

Reply:
This will be a requirement, which readme should it go in? The general one or the one specific to pcs_detection

Reply:
pcs_detection